src/applications/mhv-landing-page/constants.js

```diff
@@ -0,0 +1,91 @@
+const { entries, freeze, values } = Object;
+
+export const HEALTH_TOOL_HEADINGS = freeze({
+  APPOINTMENTS: 'Appointments',
+  MESSAGES: 'Messages',
+  MEDICATIONS: 'Medications',
+  HEALTH_RECORDS: 'Health records',
+  PAYMENTS: 'Payments',
+  MEDICAL_SUPPLIES: 'Medical supplies',
+});
+
+export const HEALTH_TOOL_NAMES = freeze(values(HEALTH_TOOL_HEADINGS));
+
+export const HEALTH_TOOL_LINKS = freeze({
+  APPOINTMENTS: freeze([
+    {
+      href: '/my-health/appointments/schedule/type-of-care',
+      text: 'Schedule a new appointment',
+    },
+    {
+      href: '/health-care/schedule-view-va-appointments/appointments',
+      text: 'Upcoming VA appointments',
+    },
+    {
+      href: '/find-locations',
+      text: 'Find VA locations',
+    },
+  ]),
+  MESSAGES: freeze([
+    {
+      href: '/my-health/secure-messages/folders/inbox',
+      text: 'Inbox',
+    },
+    {
+      href: '/my-health/secure-messages/new-message',
+      text: 'Compose message',
+    },
+    {
+      href: '/my-health/secure-messages/folders',
+      text: 'Manage folders',
+    },
+  ]),
+  MEDICATIONS: freeze([
+    {
+      href: '/my-health/medications/refill',
+      text: 'Refill VA prescriptions',
+    },
+    {
+      href: '/medications',
```

Comment:
I'm pretty sure this path should be:

```suggestion
      href: '/my-health/medications',
```

If you want to see what medications looks like (with some very fake data), you can run `yarn mock-api --responses src/applications/mhv/medications/mocks/api` and `yarn watch --env entry=mhv-landing-page,auth,medications` and visit http://localhost:3001/my-health/medications to see that medications list page locally. I don't think any allergies were mocked yet, but otherwise that's the place!

Reply:
Great catch!